File: bds_mobile/lib/core/state/user_session.dart
import 'package:bds_mobile/core/models/queue_ticket.dart';

class UserSession {
  int? userId;
  String userName = 'Guest User';
  String? email;
  String? role;
  String? token;
  int? preferredBranchId;
  QueueTicket? activeTicket;
}

final UserSession userSession = UserSession();

File: bds_mobile_rn/src/state/userSession.ts
import type { QueueTicket } from '../types/models';

export const userSession = {
  userId: null as number | null,
  userName: 'Guest User',
  email: null as string | null,
  role: null as string | null,
  token: null as string | null,
  preferredBranchId: null as number | null,
  activeTicket: null as QueueTicket | null,
};
